src/treq/test/test_response.py

```diff
@@ -98,3 +99,18 @@ def test_history_notimplemented(self):
 
     if not skip_history:
         test_history_notimplemented.skip = "History supported."
+
+    def test_fail_for_status_ok(self):
+        original = FakeResponse(200, Headers(), body=[b''])
```

Comment:
I agree, but this is inconsistent with the status codes used in the rest of the file. Change anyway?